plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/utils/ElementUtils.java

```diff
@@ -0,0 +1,279 @@
+package ru.sbtqa.tag.pagefactory.web.utils;
+
+import static java.lang.String.format;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.openqa.selenium.By;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+import ru.sbtqa.tag.pagefactory.exceptions.ElementDisabledException;
+import ru.sbtqa.tag.pagefactory.exceptions.ElementNotFoundException;
+
+public class ElementUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2sO4tUZZzFFj6Hg&open=AWfPt2sO4tUZZzFFj6Hg)

[ISSUE_KEY]: AWfPt2sO4tUZZzFFj6Hg